caddytls/setup.go

```diff
@@ -174,6 +176,13 @@ func setupTLS(c *caddy.Controller) error {
 				}
 			case "must_staple":
 				config.MustStaple = true
+			case "certificate_transparency":
+				args := c.RemainingArgs()
+				if len(args) == 0 || args[0] == "yes" {
+					config.CertificateTransparency = true
+				} else if args[0] == "no" {
+					config.CertificateTransparency = false
+				}
```

Comment:
What if we just call it `ct` instead of `certificate_transparency` (the Caddyfile is designed to be easy to write, and I think the "CT" acronym is pretty common)? Also, since it's on by default, there's no reason IMO to turn it on. The `tls` directive for example doesn't allow `tls on` -- just `tls off`. I think the only variant of this certificate_transparency line that we should allow is `ct off`.

It also means you could drop one of the test functions below.